include/benchmark/benchmark.h

```diff
@@ -444,11 +444,11 @@ struct BenchmarkInstance;
 class ThreadTimer;
 class ThreadManager;
 
-class BenchmarkTime {
+class BenchmarkTimeInfo {
```

Comment:
That is better, thanks.
I really would prefer `...Dsc`, but if @dominichamon does not share this view..

Reply:
I don't know what Dsc stands for intuitively. I don't think Info is right either and I was fine with BenchmarkTime... maybe BenchmarkTimeConverter or BenchmarkTimeOutput if you really want to pick nits.

Weak opinion, weakly held, so don't treat this as blocking.

Reply:
**D**e**sc**ription
BenchmarkTime (and _maaaybe_ BenchmarkTimeInfo) to me read as too vague,
i can't tell whether it's not supposed to be holding the actual time of the benchmark,
but the description of the measurements taken, how to interpret them, print them.
Hopefully that makes sense?

Reply:
Ah fair.. BenchmarkTimeDescription or _maybe_ BenchmarkTimeDesc then?

Reply:
Yep, any of these will be fine by me.

Reply:
Changed to `BenchmarkTimeDescription`.